panel/src/helpers/useUid.js

```diff
@@ -0,0 +1,5 @@
+let id = 0;
+
+export function useUid() {
+	return String(id++);
+}
```

Comment:
`Todo:` Check if newer Vue 3 versions don't include something like this natively.